providers/unleash/src/main/java/dev/openfeature/contrib/providers/unleash/UnleashSubscriberWrapper.java

```diff
@@ -69,7 +72,12 @@ public void toggleEvaluated(ToggleEvaluated toggleEvaluated) {
     public void togglesFetched(FeatureToggleResponse toggleResponse) {
         unleashSubscriber.togglesFetched(toggleResponse);
         if (FeatureToggleResponse.Status.CHANGED.equals(toggleResponse.getStatus())) {
+            List<String> flagsChanged = new ArrayList<>();
+            for (FeatureToggle featureToggle : toggleResponse.getToggleCollection().getFeatures()) {
```

Comment:
What does _toggleResponse.getToggleCollection().getFeatures()_ contain ?
Only changed features ?
Any documentation reference from Unleash ?

Reply:
Yeah, unfortunately, as per my other comment, it contains all the feature toggles available from the Unleash API, not just the changed ones, so this implementation doesn't work as intended.